doc/examples_sphinx-gallery/personalized_pagerank.py

```diff
@@ -0,0 +1,81 @@
+"""
+.. _tutorials-personalized_pagerank:
+
+===============================
+Personalized PageRank on a grid
+===============================
+
+This example demonstrates how to calculate and visualize personalized PageRank on a grid. We use the :meth:`igraph.GraphBase.personalized_pagerank` method, and demonstrate the effects on a grid graph.
```

Comment:
```suggestion
This example demonstrates how to calculate and visualize personalized PageRank on a grid. We use the :meth:`igraph.Graph.personalized_pagerank` method, and demonstrate the effects on a grid graph.
```

Let's link to the `Graph` version of methods, for consistency. `GraphBase` should be treated as an internal class.